src/dev/src/app/datagrid/conditional-selection/conditional-selection.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component } from '@angular/core';
+
+import { Inventory } from '../inventory/inventory';
+import { User } from '../inventory/user';
+
+@Component({
+  selector: 'clr-datagrid-conditional-selection-demo',
```

Comment:
nit: not used since its routable

Reply:
all other examples have one so didn't want to break the tradition :)